### PR TITLE
improve escape_unicode example

### PR DESCRIPTION
**Pull request by tshepang**

### CODE REVIEW

File: src/librustc_unicode/char.rs [hunk 1]
```diff
@@ -187,22 +187,16 @@ impl char {
...
     /// ```text
-    /// \
-    /// u
-    /// {
-    /// 2
-    /// 7
-    /// 6
-    /// 4
-    /// }
+    /// \u{2764}
```
Comment (steveklabnik) [inline src/librustc_unicode/char.rs]: did this change somehow?

Comment (tshepang) [inline src/librustc_unicode/char.rs]: I made it a `print!`

Comment (steveklabnik) [inline src/librustc_unicode/char.rs]: ah!

Comment (eddyb) [inline src/librustc_unicode/char.rs]: Maybe separating them would make what's happening here clearer?
For a second there I thought it printed the whole line at once.

Comment (tshepang) [inline src/librustc_unicode/char.rs]: As in `print!` is not distinct enough from `println!`?

**Pull request merged**